src/components/Utils/PrivateRouteShelter.js

```diff
@@ -5,5 +5,5 @@ import AuthService from '../../services/authService';
 export default function PrivateRouteShelter({ children }) {
```

Comment:
Maybe move this function to `PrivateRoute.js`?

`import React from 'react';
import { Navigate } from 'react-router-dom';
import AuthService from '../../services/authService';

function PrivateRoute({ children }) {
  return AuthService.isLoggedIn()
    ? children
    : <Navigate to={'/login?redirect=' + window.location.pathname} />;
}

function PrivateRouteShelter({ children }) {
  return AuthService.isLoggedIn() && AuthService.isShelterAdmin()
    ? children
    : <Navigate to={'/login/shelter?redirect=' + window.location.pathname} />;
}

export default {
  PrivateRoute,
  PrivateRouteShelter
};`